src/inputType/PanInput.ts

```diff
@@ -339,23 +341,28 @@ export class PanInput implements InputType {
   private _attachElementEvent(observer: InputTypeObserver) {
     const activeEvent = convertInputType(this.options.inputType);
     if (!activeEvent) {
-      throw new Error(
-        "There is currently no inputType available for current device. There must be at least one available inputType."
-      );
+      return;
     }
     this._observer = observer;
     this._enabled = true;
     this._activeEvent = activeEvent;
-    activeEvent?.start.forEach((event) => {
+    activeEvent.start.forEach((event) => {
       this.element?.addEventListener(event, this._onPanstart);
     });
+    // adding event listener to element prevents invalid behavior in iOS Safari
+    activeEvent.move.forEach((event) => {
+      this.element?.addEventListener(event, () => {});
+    });
```

Comment:
I also found out that adding an empty "touchmove" handler to either element / window prevents `cancelable: false` in iOS, but didn't mention it as it looks quite bad to add an empty handler like this.
Well, I hope you add a touchmove/end handler on creation, but, if it's your choice, I'd say okay about this.